api/src/main/java/io/grpc/SynchronizationContext.java

```diff
@@ -162,6 +164,12 @@ public String toString() {
     return new ScheduledHandle(runnable, future);
   }
 
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/10245")
```

Comment:
These links should point to a new issue created to track stabilizing this API. (See issues with the "experimental API" label for examples.)